ros1_bridge/__init__.py

```diff
@@ -606,7 +606,7 @@ def determine_common_services(
                 ros2_type = str(ros2_fields[direction][i].type)
                 ros1_name = ros1_field[1]
                 ros2_name = ros2_fields[direction][i].name
-                if ros1_type != ros2_type or ros1_name != ros2_name:
```

Comment:
I don't think you can just get rid of the field name checking. Rather if the field names don't match, shouldn't the code compare the issue against the mapping yaml?

Reply:
Thank you for your review, I'll check it.

Reply:
@RaAndOn

Could you help review https://github.com/ros2/ros1_bridge/pull/293/commits/873c2057ce820bd3f2c36245a54daad9ddfa860c again?